pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -86,6 +87,10 @@ const (
 	// latestFileName is the name of a file in the collection which contains the
 	// path of the most recently taken full backup in the backup collection.
 	latestFileName = "LATEST"
+
+	// latestHistoryDirectory is the directory where all 22.1 and beyond
+	// LATEST files will be stored as we no longer want to overwrite it.
+	latestHistoryDirectory = "latest-history/LATEST"
```

Comment:
nit: latestHistoryDirectory should be just that, a directory. It is the users responsibility to `path.Join` the directory with the latest file name. I'd decouple the two for clarity.